plaso/parsers/nids_fast.py

```diff
@@ -0,0 +1,327 @@
+# -*- coding: utf-8 -*-
+"""Parser for NIDS alert data in the so-called fast-log format,
+
+Fast.log format is a single line log format as shown below. There exist two
+variants:
+
+  Snort3:
+    Month/Day-Hour:Minute:Second:FractionOfSeconds [**] [sid] "string" [**] \
+    [Classification: string] [Priority : int] {protocol} SOURCE_IP:SOURCE_PORT \
+    -> DESTINATION_IP:DESTINATION_PORT
+
+  Suricata:
+    Year/Month/Day-Hour:Minute:Second:FractionOfSeconds [**] [sid] string [**] \
+    [Classification: string] [Priority : int] {protocol} SOURCE_IP:SOURCE_PORT \
+    -> DESTINATION_IP:DESTINATION_PORT
+
+More information on the format specification of Suricata can be found here:
+  https://suricata.readthedocs.io/en/suricata-6.0.0/configuration/suricata-yaml.html#line-based-alerts-log-fast-log
+
+"""
+
+import re
+
+import pyparsing
+from dfdatetime import time_elements as dfdatetime_time_elements
+from plaso.containers import events, time_events
+from plaso.lib import definitions, errors
```

Comment:
nit: single import per line

Reply:
Changed in 5759084c0311433ffd012dc583dadece99368f3f.